jpa/subsystem/src/main/java/org/jboss/as/jpa/config/PersistenceUnitMetadataImpl.java

```diff
@@ -55,6 +55,8 @@
     // required: name of the persistent unit scoped to deployment file
     private volatile String scopedName;
 
+    private volatile ArrayList<String> containingModuleName;
```

Comment:
More tracking is added to track which (sub-)module a persistence unit is defined in.